.github/workflows/update-libs.yaml

```diff
@@ -0,0 +1,13 @@
+name: Auto-update Charm Libraries
+on:
+  # Manual trigger
+  workflow_dispatch:
+  # Check regularly the upstream every twelve hours
+  schedule:
+    - cron: "0 0,12 * * *"
+
+jobs:
+  update-lib:
+    name: Check libraries
+    uses: canonical/observability/.github/workflows/charm-update-libs.yaml@main
```

Comment:
This job won't work until we set up a bot.

Reply:
fixed, change the CI like we have in the cos-server